packages/calcite-components/src/components/table/table.e2e.ts

```diff
@@ -3178,4 +3178,113 @@ describe("keyboard navigation", () => {
+            <calcite-table-cell>cell content</calcite-table-cell>
+            <calcite-table-cell>cell content</calcite-table-cell>
+          </calcite-table-row>
+          <calcite-table-row selected>
+            <calcite-table-cell>cell content 7</calcite-table-cell>
+            <calcite-table-cell>cell content</calcite-table-cell>
+            <calcite-table-cell>cell content</calcite-table-cell>
+            <calcite-table-cell>cell content</calcite-table-cell>
+          </calcite-table-row>
+          <calcite-table-row>
+            <calcite-table-cell>cell content 8</calcite-table-cell>
+            <calcite-table-cell>cell content</calcite-table-cell>
+            <calcite-table-cell>cell content</calcite-table-cell>
+            <calcite-table-cell>cell content</calcite-table-cell>
+          </calcite-table-row>
+          <calcite-table-row>
+            <calcite-table-cell>cell content 9 </calcite-table-cell>
+            <calcite-table-cell>cell content</calcite-table-cell>
+            <calcite-table-cell>cell content</calcite-table-cell>
+            <calcite-table-cell>cell content</calcite-table-cell>
+          </calcite-table-row>
+          <calcite-table-row>
+            <calcite-table-cell>cell content 10</calcite-table-cell>
+            <calcite-table-cell>cell content</calcite-table-cell>
+            <calcite-table-cell>cell content</calcite-table-cell>
+            <calcite-table-cell>cell content</calcite-table-cell>
+          </calcite-table-row>
+          <calcite-table-row>
+            <calcite-table-cell>cell content 11</calcite-table-cell>
+            <calcite-table-cell>cell content</calcite-table-cell>
+            <calcite-table-cell>cell content</calcite-table-cell>
+            <calcite-table-cell>cell content</calcite-table-cell>
+          </calcite-table-row>
+          <calcite-table-row>
+            <calcite-table-cell>cell content 12</calcite-table-cell>
+            <calcite-table-cell>cell content</calcite-table-cell>
+            <calcite-table-cell>cell content</calcite-table-cell>
+            <calcite-table-cell>cell content</calcite-table-cell>
+          </calcite-table-row>
+          <calcite-table-row>
+            <calcite-table-cell>cell content 13</calcite-table-cell>
+            <calcite-table-cell>cell content</calcite-table-cell>
+            <calcite-table-cell>cell content</calcite-table-cell>
+            <calcite-table-cell>cell content</calcite-table-cell>
+          </calcite-table-row>
+        </calcite-table>`,
+      );
+
+      const table = await page.find("calcite-table");
+
+      expect(await table.getProperty("currentPage")).toBe(2);
+
+      table.setProperty("currentPage", 3);
+      await page.waitForChanges();
+
+      expect(await table.getProperty("currentPage")).toBe(3);
+
+      const chevron = await page.find(`calcite-table >>> calcite-pagination >>> .${PAGINATION_CSS.chevron}`);
+      chevron?.click();
```

Comment:
`E2EElement#click` is async, so you'll need to await here. Currently, you're calling `click` on a promise.

Note that `E2EElement#click` also calls `page.waitForChanges` under the hood, so no need to call it right below.

Also, the E2EElement is guaranteed by `find`, so no need for `?`.